src/rust/ide/view/src/documentation.rs

```diff
@@ -56,21 +63,25 @@ enum InputFormat {
 
 pub use visualization::container::overlay;
 
+#[allow(unused_qualifications)]
+type ClickClosure = Closure<dyn std::ops::FnMut(web_sys::MouseEvent)>;
+
 /// Model of Native visualization that generates documentation for given Enso code and embeds
 /// it in a HTML container.
 #[derive(Clone,CloneRef,Debug)]
 #[allow(missing_docs)]
-pub struct ViewModel {
+pub struct Model {
     logger : Logger,
     dom    : DomSymbol,
     size   : Rc<Cell<Vector2>>,
     /// The purpose of this overlay is stop propagating mouse events under the documentation panel
     /// to EnsoGL shapes, and pass them to the DOM instead.
     overlay        : component::ShapeView<overlay::Shape>,
     display_object : display::object::Instance,
+    closures       : Rc<CloneCell<Vec<ClickClosure>>>
```

Comment:
the name "closures" is like naming a variable "variable". Names should semantically tell what fields are for.